build/mvn

```diff
@@ -69,14 +69,23 @@ install_app() {
 
 # Install maven under the build/ folder
 install_mvn() {
-  local APACHE_MIRROR=${APACHE_MIRROR:-'https://www.apache.org/dyn/closer.lua?action=download&filename='}
+  MVN_VERSION="3.3.9"
```

Comment:
Ah. For that, I tested already. When there is no maven, this script installs the correct maven. Thank you again!